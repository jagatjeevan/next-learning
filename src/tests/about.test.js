import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import About from '../../pages/about';

describe('App', () => {
  test('renders a text', () => {
    const { getByText } = render(<About />);
    expect(getByText('About Page')).toBeInTheDocument();
  });

  test('renders a snapsÌµhot', () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
