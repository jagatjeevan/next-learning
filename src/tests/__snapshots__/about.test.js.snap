// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapsÌµhot 1`] = `
<div>
  <div>
    <h1>
      About Page
    </h1>
    <h2>
      About Page 2
    </h2>
  </div>
  <div>
    <button
      onClick={[Function]}
      type="button"
    >
      Fetch data
    </button>
  </div>
</div>
`;
